### DISCUSSION TITLE
Galaxy Brain 2

### DISCUSSION THREAD

**Discussion by chaymankala** [drknzz/GitHub-Achievements#383] [Q&A] [answered]
want again

**Comment by chaymankala** [comment, marked as answer]
Is this?

**Comment by Coolbreeze151** [comment]
To get the Galaxy Brain achievement you need to answer at least 2 discussions and have the OP accept your answers.

If you would like, we can exchange with each other.

https://github.com/Schweinepriester/github-profile-achievements/discussions/945#discussioncomment-10782175

https://github.com/Schweinepriester/github-profile-achievements/discussions/946#discussioncomment-10782183

**Reply by chaymankala** [reply]
Its locked to collaborators, im unable to answer
